switch-configuration/config/scripts/switch_config_loader

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env perl
+#
+# This script is intended as a replacement for former scripts:
+#        bulk_local_load_switches
+#        override_switches
+#        update_switches
+#
+# Basic functionality of those three scripts was very similar, and the intent here is to both
+# replace the antiquated and hacky originals with a version built on the Loader.pm library
+# which is built on expect and contains the reusable portions of all of those scripts that
+# can then be improved in one place providing benefit to all these functions.
+#
+# Original funciton/intent of the initial scripts:
+#
+# bulk_local_laod_switches
+#        Built on override_switches, but optimized for loading multiple switches
+#        at a work party or similar situation.
+#
+# override_switches
+#        Send out updated configurations to switches and apply them in situ
+#        using "load override" functionality.
+#
+# update_switches
+#        A predecessor to override switches rendered obsolete by the development
+#        of override switches, long overdue for retirement.
+#
+#
+# New finctionality:
+#
+# command line usage:
```

Comment:
Would it be possible to add this comment section as a flag: `-h` for the switch_config_loader? I think that would be really helpful for anyone whose not super familiar with the tool.

Reply:
Yeah, I can add usage() and -h capabilities, that should be pretty easy.

Reply:
This has been done and pushed.